recipes/pyaztro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyaztro" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyaztro-{{ version }}.tar.gz
+  sha256: 6eb114573ef75ada94b2f82394df8dcac392382ac6a99a6f97d515069a5b8560
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
The linter failed here. We need min python versions with noarch packages.

Reply:
Oops sorry I got confused... Pushed.